Learn-CSS-Blog/index8.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div id="main">
+        <header></header>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Nest a `nav` element inside of the `<header>`.
```

Comment:
I think it's fine to remove the example here, too:

Nest a `<nav>` element inside of the `<header>`.